### PR TITLE
3 add spare node for mysql

### LINKED ISSUE #3: Add spare node for MySQL

### PR DESCRIPTION
**Pull request by Browix**
## What changes does your PR bring?

In a few words, describe what changes your commits make to the code.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (non-breaking change to the build process or auxiliary tools)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation (Improvements or additions to documentation)
- [x] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## References

Fixes:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: examples/create_mysql_database_with_sparenode/main.tf [hunk 1]
```diff
@@ -0,0 +1,17 @@
+provider "digitalocean" {
+  token = var.do_token
+}
+
+module "terraform-module-digitalocean-mysql" {
```
Comment (stawii) [inline examples/create_mysql_database_with_sparenode/main.tf]: naming

File: examples/create_mysql_database_with_sparenode/variables.tf [hunk 2]
```diff
@@ -0,0 +1,4 @@
+variable "do_token" {
```
Comment (stawii) [inline examples/create_mysql_database_with_sparenode/variables.tf:1]: default=null

File: examples/create_mysql_database_with_sparenode/versions.tf [hunk 3]
```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.5.5"
```
Comment (stawii) [inline examples/create_mysql_database_with_sparenode/versions.tf]: new line after

File: main.tf [hunk 4]
```diff
@@ -28,3 +23,24 @@ resource "digitalocean_database_cluster" "main" {
...
+
+resource "digitalocean_database_replica" "main" {
+  count      = var.replica_enable ? 1 : 0
+  cluster_id = digitalocean_database_cluster.main.id
+  name       = coalesce(var.replica_cluster_name, "${var.cluster_name}-replica")
+  size       = var.replica_node_size
+  region     = coalesce(var.replica_region, var.region)
+}
+
+resource "digitalocean_database_firewall" "replica_fw" {
```
Comment (stawii) [inline main.tf]: just `replica` seems better.

File: outputs.tf [hunk 5]
```diff
@@ -47,12 +47,22 @@ output "main_default_user_password" {
...
+
+output "replica_uuid" {
+  description = "Replica UUID."
+  value       = digitalocean_database_replica.main[*].uuid
+}
+
+output "replica_id" {
+  description = "Replica ID."
+  value       = digitalocean_database_replica.main[*].id
+}
```
Comment (stawii) [inline outputs.tf]: missing newline at EOF.

**Review by stawii — CHANGES_REQUESTED** [review]

File: examples/create_mysql_database_with_sparenode/main.tf (hunk 1 above)
Comment (Browix) [reply examples/create_mysql_database_with_sparenode/main.tf]: fixed

File: examples/create_mysql_database_with_sparenode/variables.tf (hunk 2 above)
Comment (Browix) [reply examples/create_mysql_database_with_sparenode/variables.tf:1]: fixed

File: examples/create_mysql_database_with_sparenode/versions.tf (hunk 3 above)
Comment (Browix) [reply examples/create_mysql_database_with_sparenode/versions.tf]: fixed

File: main.tf (hunk 4 above)
Comment (Browix) [reply main.tf]: fixed

File: outputs.tf (hunk 5 above)
Comment (Browix) [reply outputs.tf]: fixed

**Review by stawii — APPROVED** [review]

**Pull request merged**